plugins/armorblox/icon_armorblox/triggers/get_incidents/trigger.py

```diff
@@ -0,0 +1,32 @@
+import insightconnect_plugin_runtime
+import time
+from datetime import datetime, timedelta
+
+from .schema import GetIncidentsInput, GetIncidentsOutput, Input, Output, Component
+# Custom imports below
+
+ARMORBLOX_INCIDENT_API_TIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+ARMORBLOX_INCIDENT_API_TIME_DELTA_IN_DAYS = 1
+
+class GetIncidents(insightconnect_plugin_runtime.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='get_incidents',
+                description=Component.DESCRIPTION,
+                input=GetIncidentsInput(),
+                output=GetIncidentsOutput())
+
+    def run(self, params={}):
+        """Run the trigger"""
+        fetch_interval = int(params.get(Input.INTERVAL))
```

Comment:
I suggest adding the default value 
```suggestion
        fetch_interval = params.get(Input.INTERVAL, DEFAULT_INTERVAL_VALUE)
```